e2e/vm/host_gateway_ip_test.go

```diff
@@ -0,0 +1,58 @@
+package vm
+
+import (
+	"os/exec"
+	"path/filepath"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+	"github.com/runfinch/common-tests/command"
+	"github.com/runfinch/common-tests/option"
+
+	"github.com/runfinch/finch/e2e"
+)
+
+const defaultImage    = "public.ecr.aws/docker/library/alpine:latest"
+const nerdctlRootlessCfgPath = "/home/root.linux/.config/nerdctl/nerdctl.toml"
```

Comment:
Why need this?

Reply:
oh I thought I need to access the toml file for the host_gateway_ip value since it is update there